### PR TITLE
Add switchMania module + correct a library problem

### PR DESCRIPTION
**Pull request by 6e656d6d61**

### CODE REVIEW

File: modules/switchMania/switchMania.ino [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+#define SWITCHES_OF_NUMBER (4)
+
+#define BUTTON (2)
+#define FIRST_SWITCHE (3)
+#define SECOND_SWITCHE (6)
+#define THIRD_SWITCHE (12)
+#define FOURTH_SWITCHE (13)
+
+bool patterns[MAX_DIFFICULTY][SWITCHES_OF_NUMBER] = {{ON, OFF, ON, ON},
```
Comment (valkheim) [inline modules/switchMania/switchMania.ino]: Make use of standard HIGH / LOW

File: modules/switchMania/switchMania.ino [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/****************************************
+ * Switch Mania
+ *
+ * Léo Paolo
```
Comment (valkheim) [inline modules/switchMania/switchMania.ino]: Wrong name, isn't it ?

File: modules/switchMania/switchMania.ino [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+#include <ktne_core.h>
+
+#define ON (HIGH)
+#define OFF (LOW)
+
+#define SWITCHES_OF_NUMBER (4)
+
+#define BUTTON (2)
+#define FIRST_SWITCHE (3)
```
Comment (valkheim) [inline modules/switchMania/switchMania.ino]:
```diff
- switche
+ switch
```

File: modules/switchMania/switchMania.ino [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+void  patternValidation() {
+  if (digitalRead(FIRST_SWITCHE) == patterns[difficulty - 1][0] &&
+      digitalRead(SECOND_SWITCHE) == patterns[difficulty - 1][1] &&
+      digitalRead(THIRD_SWITCHE) == patterns[difficulty - 1][2] &&
+      digitalRead(FOURTH_SWITCHE) == patterns[difficulty - 1][3]) {
+        defuseModule();
+      }
+   else {
+    applyPenality(20);
```
Comment (valkheim) [inline modules/switchMania/switchMania.ino]: Penality is the same on each difficulty level ?
Maybe we could `applyPenality(20 + (difficulty - 1) * 5)` or anything like that.

**Review by valkheim — CHANGES_REQUESTED** [review]

**Review by valkheim — APPROVED** [review]

**Comment by 6e656d6d61** [comment]
Need to squash ?

**Comment by valkheim** [comment]
That would be fine. Deleting the empty last line / avoid using golang curly-braces style would be fine too.
Will be merged if other guys are ok with your code.

**Comment by 6e656d6d61** [comment]
Everything has been corrected

**Pull request merged**